core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -379,84 +387,40 @@ void CmpSeabaseDDL::createSeabaseTableLike(
 }
 
 // ----------------------------------------------------------------------------
-// Method: createSeabaseTableExternalHive
+// Method: createSeabaseTableExternal
```

Comment:
Thanks. This will be a very useful methods during PoCs. Maybe the bulk loader could take a create option and use this method to create an empty Traf table before loading into it.